server/models/dtos/project_dto.py

```diff
@@ -89,6 +98,8 @@ class ProjectDTO(Model):
     priority_areas = BaseType(serialized_name='priorityAreas')
     last_updated = DateTimeType(serialized_name='lastUpdated')
     author = StringType()
+    task_creation_mode = StringType(required=True, serialized_name='taskCreationMode',
```

Comment:
I'm so sorry to bother but I'm still unable to reproduce the bug you are talking about. I may be stupid but I can always edit a project after it has been created.
Can you please describe the complete workflow (like if I was 5-years old)?